### ISSUE TITLE
Craft 5 support

### ISSUE THREAD

**Issue by thomasvantuycom** [sjelfull/craft3-imgix#35]
Are there any plans to port this plugin to Craft 5?

**Comment by sjelfull** [comment]
There is, although no ETA yet.

**Comment by joshuapease** [comment]
Hi @sjelfull! Hope you're doing well. 

Are you open to a PR to migrate this plugin to Craft 5? 

We have a Craft 5 upgrade looming for a large site, and I think it will be simpler for us to update this plugin instead of migrating to a different Imgix plugin (such as Imager X).

**Comment by sjelfull** [comment]
@joshuapease that would be fantastic, please go ahead!

**Comment by bryantAXS** [comment]
Possible to pull in those PR and get it merged in?

**Comment by sjelfull** [comment]
@bryantAXS Yes, will merge and release once @joshuapease confirms its ready for review.

**Comment by joshuapease** [comment]
Ready for review! Thanks for the friendly nudge. 

- https://github.com/sjelfull/craft3-imgix/pull/36

**Comment by anchovy** [comment]
Hi 👋 — thanks for merging the Craft 5 compatibility update, that’s hugely appreciated!

Would it be possible to tag a new release (e.g. 3.1.0 or similar) so Composer can pull in the Craft 5-compatible version without having to use dev-master? Right now the latest tagged version is still 3.0.0, so our upgrade process gets stuck on the older release.

Thanks again for maintaining this plugin 🙏

**Comment by sjelfull** [comment]
Sorry for the long wait, 4.0.0 with Craft 5 support is now out.
